soh/soh/Enhancements/randomizer/ShufflePots.cpp

```diff
@@ -0,0 +1,99 @@
+#include "ShufflePots.h"
+#include "soh_assets.h"
+
+extern "C" {
+#include "variables.h"
+#include "overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.h"
+#include "overlays/actors/ovl_Door_Shutter/z_door_shutter.h"
+
+u8 Randomizer_GetSettingValue(RandomizerSettingKey randoSettingKey);
```

Comment:
This is a method exported for .c access to the rando context.
Within .cpp we can and should defer to accessing the rando context directly. `hook_handlers` and other places have used macros like
```
#define RAND_GET_OPTION(option) Rando::Context::GetInstance()->GetOption(option).GetContextOptionIndex()
```
We don't have a convenient location to house these yet, so its just been duplicated a couple times.

Reply:
I won't macro it in this file as it's only used twice, but agree it's not needed anymore 👍